package.json

```diff
@@ -42,5 +42,5 @@
     "Ognian Tschakalov <ognian.tschakalov@ogi-it.com>",
     "Simon Wade <simon.wade@gmail.com>"
   ],
-  "license": "BSD"
+  "license": "BSD-3-Clause"
```

Comment:
This would probably be a good thing for @villadora to weigh in on, especially considering that they are mostly hands off of this lib at this point. The [4-Clause](https://tldrlegal.com/license/4-clause-bsd) requires attribution (which I doubt most people follow). The [3-Clause](https://www.tldrlegal.com/l/bsd3) is an OSI approved license and would simplify things for users while still maintaining the same licensing protections.